example_test.py

```diff
@@ -18,6 +18,13 @@
 """
 Test the :py:class`pyfakefs.example` module to demonstrate the usage of the
 :py:class`pyfakefs.fake_filesystem_unittest.TestCase` base class.
+
+Fake filesystem functions like CreateFile(), CreateDirectory() or CreateLink()
```

Comment:
I played around with sphinx (haven't used it before) and created the docs with the current state on github pages: http://mrbean-bremen.github.io/pyfakefs. Not sure if this is the way to go (e.g. autogenerate docs on gh-pages), but it helps to get some preview.